about.html

```diff
@@ -0,0 +1,130 @@
+            <a href="#" class="logo">JA KENYA</a>
+
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li><a href="#" class="nav-link">CC Campaign</a></li>
+                
+            </ul>
+
+           
+
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+        </div>
+        </nav>
+    </header>
+
+    <section class="hero-about">
+        <div class="about-hero-wrapper">
+        <p class="hello">"Hello Young Nation</p>
+        <h1>JUNIOR  ACHIEVER <br />ALL SUMMIT 2015</h1>
+        </div>
+        
+        <p class="about-description">The National Summit brings together the community of  
+            young mindsets eager and ready to learn. Young people are then able 
+        to pitch their ideas to show innovation, Creativity in this fast ever changing 
+    Tech world</p>
+
+        <div class="support">
+        <p class="message">Please contact us per Email for nay further questions about JA Kenya Summit 2023!</p>
+        <p class="email">ja2320@kenya.org </p>
+
+        </div>
+    
+    </section>
+    <section class="logo-sub">
+        <h2>JA Launch <br /> National Summit 2023 logo</h2>
+        <hr />
+        <p>The logo of JA Kenya 2023 was decided through the logo competition from 8, June to 7, July.</p>
+        <hr />
+        <i class="uil uil-wind-sun logo-design"></i>
+        <hr class="d-line"/>
+
+        <div class="past-summits">
+            <h2>See the past JA Kenya Summits</h2>
+            <hr />
+            <p>Take a look at the past two
+             JA National Summits which took place in Turkana and Nairobi</p>
+             <div class="all-testimonials">
+             <div class="testimonial summit-1">
+             <h3 class="year">2011</h3>
+             <p class="about-summit">JA Summit 2011 in Turkana</p>
+            </div>
+
+             <div class="testimonial">
+             <h3 class="year">2011</h3>
+             <p class="about-summit">JA Summit 2011 in Nairobi</p>
+            </div>
+
+        </div>
+             
+        </div>
+
+    </section>
+
+    <section class="partner">
+        <h2>Partner</h2>
+        <hr />
+        <div class="partner-logos">
+            <img src="./images/airbnb.svg" alt="">
+            <img src="./images/daumkakaologo.svg" alt="">
+            <img src="./images/googlekorea.svg" alt="">
+            <img src="./images/mozilla.svg" alt="">
+            <img src="./images/naver.svg" alt="">
```

Comment:
[OPTIONAL] To make your page more semantic, I suggest you add a descriptive value to your `alt` attribute👍. This gives screen readers an idea of what the image is about when presenting it to visually challenged people and also can serve as a means to understand what the image is about when the link to the image is broken or there's a bad network👍